en/chromebook_setup/instructions.md

```diff
@@ -2,33 +2,40 @@ You can [skip right over this section](http://tutorial.djangogirls.org/en/instal
 are, your installation experience will be a little different. You can ignore the
 rest of the installation instructions.
 
-### Cloud 9
+### CodeAnywhere
 
-Cloud 9 is a tool that gives you a code editor and access to a computer running
+CodeAnywhere is a tool that gives you a code editor and access to a computer running
 on the Internet where you can install, write, and run the software. For the duration
-of the tutorial, Cloud 9 will act as your _local machine_. You'll still be
+of the tutorial, CodeAnywhere will act as your _local computer_. You'll still be
 running commands in a terminal interface just like your classmates on OS X,
 Ubuntu, or Windows, but your terminal will be connected to a computer running
-somewhere else that Cloud 9 sets up for you.
+somewhere else that CodeAnywhere sets up for you.
 
-1. Install Cloud 9 from the [Chrome web store](https://chrome.google.com/webstore/detail/cloud9/nbdmccoknlfggadpfkmcpnamfnbkmkcp)
-2. Go to [c9.io](https://c9.io)
-3. Sign up for an account
-4. Click _Create a New Workspace_
-5. Name it _django-girls_
-6. Select the _Blank_ (second from the right on the bottom row with orange logo)
+1. Go to [codeanywhere.com](https://codeanywhere.com)
+2. Sign up for an account
+3. Check your email and verify your account
+4. Close the connection wizard (We'll go back to that in a second)
+3. Click _File -> New Project_
+4. Name it _django-girls_
+5. In the Connection Wizard, name your connection _tutorial-container_
+6. Search for "python" in the search box
+7. Select _Django - Ubuntu 14.04_
+8. Wait for your container to start. This should take 1-2 minutes.
 
-Now you should see an interface with a sidebar, a big main window with some
-text, and a small window at the bottom that looks something like this:
 
-{% filename %}Cloud 9{% endfilename %}
+Now you should see an interface with a menu bar, a sidebar, and a big main
+window with some text. We want to open a _terminal_, a window that will let us
+give instructions to the computer CodeAnywhere has started for us.
+
+To open the terminal, right click on "tutorial-container" in the sidebar and
+select _SSH Terminal_. The main area should now show a prompt that looks like this:
```

Comment:
With the container just created, there are 3 things called "tutorial-container" in the sidebar for me:

<img width="667" alt="codeanywhere-sshterminal" src="https://user-images.githubusercontent.com/877585/32405747-e7625a2e-c173-11e7-9ce4-f0a7713cabb9.png">

The first tab seems to already be an SSH terminal – should the tutorial point to that instead? Otherwise it should probably be specified `right click on "tutorial-container" underneath "Connections" in the sidebar.